.storybook/main.css

```diff
@@ -76,6 +76,10 @@
   .text-4 {
     @apply text-xs font-medium;
   }
+  
+  .text-5 {
+    @apply text-blue-400 font-bold text-xs;
```

Comment:
Remove 'text-blue-400' and add this directly into the component.
These text styles should be very reusable, so font-weight and font-size is really enough.